src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/DigitalWarrantResourceImplIntTest.java

```diff
@@ -0,0 +1,239 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpMethod;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.justice.hmpps.prison.api.model.digitalwarrant.CourtCase;
+import uk.gov.justice.hmpps.prison.api.model.digitalwarrant.Offence;
+import uk.gov.justice.hmpps.prison.api.model.digitalwarrant.Sentence;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderCharge;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderCourtCase;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderSentence;
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderBookingRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderChargeRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderCourtCaseRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderSentenceRepository;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+public class DigitalWarrantResourceImplIntTest extends ResourceTest {
+    @Autowired
+    private OffenderCourtCaseRepository offenderCourtCaseRepository;
```

Comment:
Minor observations; we would prefer to use webTestClient going forward for new test. Also if possible try creating a offender / booking "on the fly" rather than using seed data. Tests tend to read better that way

Reply:
Thanks I'll make a note of this on the improvement ticket.